src/components/CancelPausePopup/CancelPausePopup.js

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import InnerPopupWrapper from 'components/InnerPopupWrapper';
+import { withTranslation } from 'react-i18next';
+import labeling from 'containers/labeling';
+import Button from 'components/Button';
+import Loader from 'components/Loader';
+import { updateSwitch } from 'api';
+import checkmarkIconBase from 'assets/images/checkmarkBase';
+
+import {
+  ContentStyled,
+  TitleStyled,
+  TextStyled,
+  ButtonWrapperStyled
+} from 'components/InnerPopupWrapper/InnerPopupWrapperStyled';
+
+const CancelPausePopup = ({
+  popupData: { pendingSwitchId, baseOfferExpirationDate, baseOfferPrice },
+  hideInnerPopup,
+  updateList,
+  setSwitchDetails,
+  t
+}) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [step, setStep] = useState(1);
+
+  const planDetailsState = useSelector(state => state.planDetails);
+  const switchDetails = planDetailsState.switchDetails[pendingSwitchId];
```

Comment:
what if any of these (planDetailsState  or planDetailsState.switchDetails) won't exist?

Reply:
I dont think that there is any possibility when those values are not available, because this popup is opened from infobox about pending switch so all information should be in store already

Reply:
Okey. There is also a way to add default value when t() function is used with key:
https://www.i18next.com/translation-function/essentials#passing-a-default-value